resin/resin/core/window.cpp

```diff
@@ -175,5 +175,114 @@ void Window::imgui_setup() const {
   ImGui_ImplGlfw_InitForOpenGL(window_ptr_, true);
   ImGui_ImplOpenGL3_Init("#version 150");
 }
+void Window::imgui_set_style() {
+  ImGuiIO& io = ImGui::GetIO();
+
+  std::filesystem::path path = std::filesystem::current_path() / "assets/fonts";
```

Comment:
unimportant side note: we need to use `argv[0]` for all assets, as it's annoying that the assets folder must be in the current working directory, created a separate task for it (SDF-104)